### PR TITLE
Support Auto-Discovery

### PR DESCRIPTION
**Pull request by m1guelpf**
This will make the package work with [auto-discovery](https://github.com/laravel/framework/pull/19420) in Laravel 5.5

<details>
<summary>Boring legal stuff</summary>

- I agree to assign the copyright of my contribution to Dev7studios.

- I declare that the contribution is in the public domain.
</details>

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -25,5 +25,11 @@
         "psr-4": {
             "Tests\\Laravel\\": "tests/laravel/"
         }
+    },
+    "extra": {
+    "laravel": {
+        "providers": [
+            "Dev7studios\\Chip\\Laravel\\ChipServiceProvider"
+        ]
```
Comment (gilbitron) [inline composer.json]: Are you missing a closing `}`?

Comment (m1guelpf) [reply composer.json]: Yep, fixed. Thank you!

**Comment by gilbitron** [comment]
Thanks. As per our [contibution guidelines](https://github.com/Dev7studios/chip/blob/master/CONTRIBUTING.md) you need to:

* Agree to assign the copyright of your contribution to Dev7studios (in your commit message or pull request).
* Declare that the contribution is in the public domain (in your commit message or pull request).

**Comment by m1guelpf** [comment]
@gilbitron Added!

**Comment by gilbitron** [comment]
Thanks 👍

**Pull request merged**